src/Tagger/RainbowTaggerProvider.cs

```diff
@@ -29,7 +29,7 @@ public class CreationListener : IViewTaggerProvider
                 ITagAggregator<IClassificationTag> aggregator = _aggregator.CreateTagAggregator<IClassificationTag>(textView);
                 _isProcessing = false;
 
-                return textView.Properties.GetOrCreateSingletonProperty(() => new RainbowTagger(textView, _registry, aggregator)) as ITagger<T>;
+                return buffer.Properties.GetOrCreateSingletonProperty(() => new RainbowTagger(textView, buffer, _registry, aggregator)) as ITagger<T>;
```

Comment:
I think if you open a XAML file the this will throw because buffer is null. May be the same for ASPX pages in split view (designer and code split). Can you verify that it works with XAML?

Reply:
I've tried both XAML and ASPX in split view. In both cases buffer wasn't null.